syntax = "proto3";

package response_models;

message RespawnPlayerResponseModel {
	int32 player_id = 1;
}